views/shared/menu.tmpl

```diff
@@ -1,13 +1,13 @@
 <ul class="qor-menu">
-  {{range .Result}}
-    {{if .SubMenus}}
-      <li qor-icon-name="{{.Name}}" {{if .Active}}class="active"{{end}}>
-        <a href="{{if .URL}}{{.URL}}{{else}}javascript:void(0);{{end}}">{{t (printf "qor_admin.menus.%v" .Name) .Name}}</a>
-        {{render "shared/menu" .SubMenus}}
+  {{range $key, $value := .Result}}
```

Comment:
Use `$_` instead of `$key` would be better since it is not used.

Reply:
yes